src/SSW.SophieBot/components/SSWSophieBot.HttpClientComponents.PersonQuery/Actions/GetEmployeesProfileAction.cs

```diff
@@ -0,0 +1,66 @@
+﻿using AdaptiveExpressions.Properties;
+using Microsoft.Bot.Builder.Dialogs;
+using Newtonsoft.Json;
+using SSWSophieBot.Components;
+using SSWSophieBot.Components.Actions;
+using SSWSophieBot.HttpClientAction.Models;
+using SSWSophieBot.HttpClientComponents.PersonQuery.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SSWSophieBot.HttpClientComponents.PersonQuery.Actions
+{
+    public class GetEmployeesProfileAction : ActionBase
```

Comment:
I suggest to use "GetBookingProfileAction" as the action name, to be more specific and avoid ambiguity.
- [ ] Change related names

Reply:
Renamed to `GetProfileWithStatusAction`